react/InfosCarrousel/styles.styl

```diff
@@ -0,0 +1,31 @@
+@require 'settings/palette'
+
+.InfosCarrousel
+    position relative
+
+    &.InfosCarrousel--primary .InfosCarrousel-arrow,
+    &.InfosCarrousel--secondary .InfosCarrousel-arrow
+        color var(--coolGrey)
+
+    &.InfosCarrousel--danger .InfosCarrousel-arrow
+        color var(--pomegranate)
```

Comment:
We have to keep this pattern in mind : theme dependent buttons. I think this could be extracted at some point.

Reply:
Those buttons could use a ThemeContext, like discussed in https://github.com/cozy/cozy-ui/pull/1302#discussion_r359225774